lib/task.js

```diff
@@ -39,14 +39,23 @@ const runBrowserify = opts => runTask('Browserify', (resolve, reject) => {
 });
 
 const runBabel = (code, opts) => runTask('Babel', (resolve, reject) => {
-	const configItem = babel.createConfigItem(babelEnvPreset);
-	opts = {presets: [configItem], compact: false};
-
-	babel.transformAsync(code, opts).then(result => {
-		resolve(result.code);
-	}).catch(err => {
-		reject(err);
-	});
+	const babelrcPath = opts.babelConfig;
+	if (babelrcPath) {
+		opts = {
+			filename: babelrcPath
```

Comment:
According to the [documentation](https://babeljs.io/docs/en/options#filename), `filename` is supposed to be the name of the compiled file.

Reply:
Well, I guess it's a way of making it work...

Reply:
I changed it to [configFile](https://babeljs.io/docs/en/options#configfile), added babelrc to false, and also updated test cases. can you check again?

Reply:
I wanted to write configFile before, there is note below the doc saying is not recommended using with babelrc true, so I set babelrc to false.

Reply:
The docs says 
> NOTE: This option does not affect loading of .babelrc files, so while it may be tempting to do configFile: "./foo/.babelrc", it is not recommended.
I honestly don't know how to do, I'll check and tell you later

Reply:
yup, exactly, I was confused too. that's why I set babelrc to false.